php8.2/Dockerfile

```diff
@@ -1,7 +1,4 @@
-# Hopefully for a short while, we are copying the root `../Dockerfile` and
-# building otherwise the same as 8.1. Delete this as soon as stable Mautic
-# supports newer!
-FROM php:8.0-apache-buster
+FROM php:8.2-apache-bookworm
```

Comment:
this file is a copy of https://github.com/thebiggive/docker-php/blob/b8e9c544addee6716c42a99d6e27368a11638f8c/Dockerfile#L11-L10 , just changed 8.1-buster to 8.2-bookworm